charter/index.html

```diff
@@ -168,19 +169,21 @@ <h1 id="title">PROPOSED Solid Working Group Charter</h1>
         <h2>Motivation and Background</h2>
         <p>It is common for web service providers to require users to surrender control over their own data, and store them in a way that is often restricted to certain systems, or can only be accessed by the applications that created it.</p>
 
-        <p>Solid extends open web standards to give priority to individual and community autonomy, allowing them to keep authority over their identity, data, and privacy, and giving them the freedom to select applications and services that suit their needs.</p>
+        <p>Solid is largely based on existing open web standards, and extends them to give priority to individual and community autonomy. It allows them to keep authority over their identity, data, and privacy, and gives them the freedom to select applications and services that suit their needs.</p>
 
         <p>Solid defines the concept of storage (one feature of the construct informally referred to as Solid Pods), in which users place data and control access to it, and a suite of protocols for applications to use storages.</p>
 
-        <p>Solid offers several advantages over more traditional architectures for data use by Web services today, including:</p>
+        <p>Solid offers several advantages over more traditional architectures for data use by web services today, including:</p>
 
         <ul>
           <li>Storage is controlled by individuals or organizations, and can be delegated to other parties.</li>
           <li>Interoperability between applications and various storage server implementations of the protocol.</li>
-          <li>A decentralized trust model enabling authentication and authorization mechanisms to operate at Web scale.</li>
+          <li>A decentralized trust model enabling authentication and authorization mechanisms to operate at web scale.</li>
```

Comment:
"web scale" is a buzzword / vague or overhyped... if we are making changes in the PR, can we bring it down a notch:

```suggestion
          <li>A decentralized trust model enabling authentication and authorization mechanisms across the web.</li>
```

On second read, I find the whole bullet point overloaded.. "trust model", really?

Reply:
> "trust model", really?

Any better suggestion?